src/dbnode/storage/block/block.go

```diff
@@ -186,26 +167,6 @@ func (b *dbBlock) Checksum() (uint32, error) {
 	return b.checksum, nil
 }
 
-func (b *dbBlock) OnRetrieveBlock(
```

Comment:
@justinjc Now that you've deleted this method, how does the seriesID get set? I think you might need to make seriesID a required argument for the `Reset` method otherwise it will mess up the LRU caching policy

Reply:
As discussed offline, going to make a separate `ResetFromDisk` function that's basically `Reset` with setting the `seriesID`